source/module_base/global_variable.h

```diff
@@ -282,6 +282,15 @@ extern std::string chg_extrap;
 extern int out_pot;
 
 extern std::string init_chg; //  output charge if out_chg > 0, and output every "out_chg" elec step.
+/// @brief method to initialize wavefunction
+/// @author kirk0830, 20230920
+extern std::string init_wfc; 
+/// @brief whether use the new psi initializer to initialize psi
+/// @author ykhuang, 20230920
+extern bool psi_initializer;
```

Comment:
How to choose the new or old method to initialize psi?

Reply:
> I don't think the old version is helpful is the new version is good enough.

That is what the most strange. By default I didnt change anything in main workflow, the psi_initializer is off by default, therefore it is incredible to see fail to pass integrated test...

Reply:
> I don't think the old version is helpful is the new version is good enough.

I have find the reason, it is because the not-fully compatible workflow between the old and this new initializer. I add instructions on this PR. 
The old version can support SDFT related calculation, but for the new one it cannot presently